ee/codegen/src/__test__/helpers/node-data-factories.ts

```diff
@@ -560,19 +560,27 @@ export function promptDeploymentNodeDataFactory(): PromptNode {
 }
 
 export function templatingNodeFactory({
```

Comment:
how do we feel about introducing a genericNodeFactory or a baseNodeFactory for building out most of these tests? Had to edit this factory bc two templating nodes with the same id/handle ids/labels would throw errors

Reply:
Seems like a good idea to me!